desktop/assets/categories.coffee

```diff
@@ -13,7 +13,7 @@ routes =
 
   '''
   /tag
-  ''': require('../apps/tag/client.coffee').init
+  ''': require('../apps/tag/client.js').default.init
```

Comment:
Is there a need to include file extensions when using CoffeeScript? With vanilla JS there’s no need as extensions are registered with `require`.